src/components/memes/MemesCard.jsx

```diff
@@ -0,0 +1,34 @@
+import propTypes from 'prop-types';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import classes from './MemesCard.module.css';
+
+const MemesCard = ({ name, image, boxCount }) => (
+  <div className={classes.memes_container__card}>
+    <h1 className={classes.forward__icon}>
+      <BsArrowRightCircle />
+    </h1>
+    <img src={image} alt="memes" className={classes.image__itm} />
+    <h1 className={classes.memes__name}>
+      Name:
+      <span>{name}</span>
+    </h1>
+    <p className={classes.box__content}>
+      Box Count:
+      <span>{boxCount}</span>
+    </p>
```

Comment:
[OPTIONAL] It would be best if you right-aligned the category name and stats as in the design guidelines.

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/9d3136d9-9dfe-4e3c-ab6d-d2d9704a3b16)

Design:

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/97aeb1dc-cae7-4402-8abb-e2883eaae73d)